WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -72,6 +72,13 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
         }
     }
 
+    /// Tracks whether the webview has called -didFinish:navigation
+    private var isLoadingWebView = true
+
+    /// If the scroll(to:) has been called before the webview content has been loaded
+    /// the hash will be stored in this variable, and scrolled to once the content is loaded
+    private var pendingHashString: String? = nil
```

Comment:
I know this is such a simple logic (and the scroll) but I wonder if that shouldn't be in the coordinator (which is responsible for all the logic) — while the view simple do what the coordinator commands.

Something like: webView load -> tell coordinator -> coordinator tell VC to scroll if needed.

Reply:
Done in ee35f5cc93eecd4588b5b6ba05e35a474526e9e0